### PR TITLE
Update Weekly Outlines for 1708

### PR DESCRIPTION
**Pull request by VictoriaVasys**
* Update projects (Night Writer instead of Enigma) 
* Include PD links under Lessons
* Add homework sections to all weeks
* Make titles more consistent
* Add Week 6 Outline

### CODE REVIEW

**Comment by VictoriaVasys** [comment]
@AliSchlereth & @neight-allen could you take a look at the changes I made & let me know if anything needs to be re-done?

File: module1/weekly_outlines/week_1.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ layout: page
 * [Flow Control](../lessons/flow_control) ([Video](https://www.youtube.com/watch?v=iZkQWR9_RpY))
 * [Debugging with Pry](../lessons/debugging_with_pry)
 * [Introducing Arrays](../lessons/intro_to_arrays_arts_and_crafts) ([Video](https://www.youtube.com/watch?v=nlwU1YtQ9SU))
+* [Exercise: Bad Connection](../homework/bad_connection)
```
Comment (AliSchlereth) [inline module1/weekly_outlines/week_1.md]: Let's take this link out. We want the exercises to be a timed code challenge type of experience.

**Comment by neight-allen** [comment]
Thanks for taking care of this!

**Comment by VictoriaVasys** [comment]
Yup, no prob. I'll close this PR; adding a new one with Bad Connection removed & Gear-up links added.

**Pull request closed without merging**